deployments/liqo_chart/crds/config.liqo.io_clusterconfigs.yaml

```diff
@@ -178,16 +180,26 @@ spec:
             liqonetConfig:
               properties:
                 podCIDR:
+                  description: the subnet used by the cluster for the pods, in CIDR
+                    notation
                   type: string
                 reservedSubnets:
-                  description: contains a list of reserved subnets in CIDR notation
-                    used by the k8s cluster like the podCIDR and ClusterCIDR
+                  description: this field is used by the IPAM embedded in the tunnelEndpointCreator
+                    if the podCIDR of a peering cluster needs to be NATed a new subnet
+                    from the 10.0.0.0/8 is used. if subnets belonging to that range
```

Comment:
```suggestion
                    from the 10.0.0.0/8 is used. If subnets belonging to that range
```

Reply:
@alacuku can you check if the new description of this field is ok? thanks!